CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ Current
 
 ### Added:
 
+- [Implement Query Split Logging](https://github.com/yahoo/fili/pull/537)
+    * Include metrics in logging to allow for better evaluation of the impact of caching for split queries.
+        - Currently there is only a binary flag (`BardQueryInfo.cached`) that is inconsistently set for split queries
+        - Three new metrics are added
+            1. Number of split queries satisfied by cache
+            2. Number of split queries not satisfied by cache
```

Comment:
Since caching is not fundamental, how about #2 read "queries actually sent to the fact store. (not satisfied by cache)"'

Reply:
Sounds good